### ISSUE TITLE
How to make my own dataset

### ISSUE THREAD

**Issue by ZDLLLL** [Kolkir/Coarse_LoFTR_TRT#2]
Hello, I would like to ask how do I create my own dataset, what should be the size of the training samples, and how do I get the depth map of the image?

**Comment by Kolkir** [comment]
Hello, you usually need some type of stereo pair camera to create such kind of dataset. Please read the [original paper](https://arxiv.org/abs/1911.10127) that describes how it was created, and there are additional information in the corresponding [repository](https://github.com/YoYo000/BlendedMVS)

**Comment by trand2k** [comment]
> Hello, you usually need some type of stereo pair camera to create such kind of dataset. Please read the [original paper](https://arxiv.org/abs/1911.10127) that describes how it was created, and there are additional information in the corresponding [repository](https://github.com/YoYo000/BlendedMVS)

Hi author,
if i dont have stereo image , can i label each pair of 2 image for trainning dataset ?
thanks for your repo.

**Comment by Kolkir** [comment]
Hi, @trand2k if have your own image pairs(two images of the same scene from different view points) you can label them by your own method, but just use the required data format.

**Comment by trand2k** [comment]
> Hi, @trand2k if have your own image pairs(two images of the same scene from different view points) you can label them by your own method, but just use the required data format.

I have some struggle when understand Fine-level Supervision loss function, this seemly is L2 norm of ground truth point and predict point, but i dont under stand how to create ground truth from depth camera, can you explain for me?
if i label pair of point in 2 image, have some area can miss in Fine-level Supervision, Does it affect the results?
thanks for your response
